app_dart/lib/src/service/branch_service.dart

```diff
@@ -160,8 +164,8 @@ class BranchService {
       },
       {
         "branch": latestCandidateBranch,
-        "name": "latestCandidateBranch",
-      }
+        "name": "dev",
+      },
```

Comment:
Aren't we over engineering this solution? why don't we just run a query on the Checklist entity?

Reply:
What type of query do you imagine? We could simplify this with enums to dedupe the code, but it's still going to require lookups to GitHub to find what RC maps to beta and stable. Files from GitHub should be aggressively cached by Cocoon, so this should be a cheap lookup.